Simplenote/Classes/NotesListFilter.swift

```diff
@@ -44,7 +44,7 @@ extension NotesListFilter {
         case .everything:
             return NSLocalizedString("All Notes", comment: "Title: No filters applied")
         case .deleted:
-            return NSLocalizedString("Trash-noun", comment: "Title: Trash Tag is selected")
+            return NSLocalizedString("Trash", comment: "Title: Trash Tag is selected")
```

Comment:
I picked this and all the other values from the translations that were set in the `.strings` file itself before the tooling fix. 

You can verify them from [this compare view](https://github.com/Automattic/simplenote-ios/compare/develop...3025190a3fe031a5bc2c34d5fe94b4e782fef021#diff-b008231bd522825760f4af589e37ded40359d336760494c92cfe707ef0417429).